docs/sphinx/developers/components.txt

```diff
@@ -11,12 +11,31 @@ can also be built with Maven by running :command:`mvn` in the component's
 subdirectory.  The Maven module name for each component (as it is shown in
 most IDEs) is also noted in parenthesis.
 
+External components
+-------------------
+
+.. _ome-common:
+
+ome-common (Common):
+
+Provides I/O classes that unify reading from files on disk, streams or files
+in memory, compressed streams, and non-file URLs.  The primary entry points
+are :javadoc:`Location <loci/common/Location.html>`,
+:javadoc:`RandomAccessInputStream <loci/common/RandomAccessInputStream.html>`
+(for reading), and :javadoc:`RandomAccessOutputStream
+<loci/common/RandomAccessOutputStream.html>` (for writing).
+
+In addition to I/O, there are several classes to assist in working with XML
+(:javadoc:`XMLTools <loci/common/xml/XMLTools.html>`), date/timestamps
```

Comment:
Having double checked, the Javadoc does no aggregate any dependency Javadoc incl. both external and OME dependencies. It should be possible to link to the `ome-common` Javadoc as we already do for e.g. https://docs.oracle.com/javase/8/docs/api/.